tests/virtctl/create_preference.go

```diff
@@ -0,0 +1,115 @@
+package virtctl
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	instancetypev1alpha2 "kubevirt.io/api/instancetype/v1alpha2"
+	generatedscheme "kubevirt.io/client-go/generated/kubevirt/clientset/versioned/scheme"
+	"kubevirt.io/client-go/kubecli"
+
+	. "kubevirt.io/kubevirt/pkg/virtctl/create/preference"
+	"kubevirt.io/kubevirt/tests/clientcmd"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+var _ = Describe("[sig-compute] create preference", decorators.SigCompute, func() {
+	var virtClient kubecli.KubevirtClient
+	var clusterPreference *instancetypev1alpha2.VirtualMachineClusterPreference
+
+	BeforeEach(func() {
+		var err error
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		if clusterPreference != nil {
```

Comment:
Where does the namespaced VMPs are deleted? Perhaps you need to clean them in [`CleanNamespaces()`](https://github.com/kubevirt/kubevirt/blob/3bee1de7b4f8bab059817cf44d636d2521dec8a9/tests/testsuite/namespace.go#L73)